src/applications/personalization/dashboard-2/components/claims-and-appeals/Appeal.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import moment from 'moment';
+
+import {
+  APPEAL_TYPES,
+  EVENT_TYPES,
+  getTypeName,
+  getStatusContents,
+  programAreaMap,
+} from '~/applications/claims-status/utils/appeals-v2-helpers';
+
+const capitalizeFirstLetter = input => {
+  const capitalizedFirstLetter = input[0].toUpperCase();
+  return `${capitalizedFirstLetter}${input.slice(1)}`;
+};
+
+const Appeal = ({ appeal, name }) => {
+  let requestEventType;
```

Comment:
most of this was pulled from https://github.com/department-of-veterans-affairs/vets-website/blob/9b24ba9b36179cd37122dd61351c815fa3125c4a/src/applications/claims-status/components/appeals-v2/AppealListItemV2.jsx